cves/kernel/CVE-2017-18017.yml

```diff
@@ -151,10 +153,11 @@ discovered:
 
     If there is no evidence as to how this vulnerability was found, then please
     explain where you looked.
-  answer:
-  automated:
-  contest:
-  developer:
+  answer: |
+    Someone shared a KASAN report to the development team. These reports show errors in memory.
```

Comment:
Can you describe KASAN a little bit more?

Reply:
Added KASAN and discoverer's search process details.